hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleAccumulator.java

```diff
@@ -274,11 +314,150 @@ yield shouldThrottleEthTxn(
+                    if (implicitCreationsCount > 0) {
+                        return shouldThrottleImplicitCreations(implicitCreationsCount, now);
+                    }
+                }
+            }
+            return !manager.allReqsMetAt(now);
+        } else {
+            if (!manager.allReqsMetAt(now)) {
+                return true;
+            }
+
+            // only check deeply if the schedule could immediately execute
+            if ((!scheduleCreate.waitForExpiry()) && (throttleType == FRONTEND_THROTTLE)) {
+                var effectivePayer = scheduleCreate.hasPayerAccountID()
+                        ? scheduleCreate.payerAccountID()
+                        : txnBody.transactionID().accountID();
+
+                final var innerTxnInfo = new TransactionInfo(
+                        Transaction.DEFAULT,
+                        innerTxn,
+                        TransactionID.DEFAULT,
+                        effectivePayer,
+                        SignatureMap.DEFAULT,
+                        Bytes.EMPTY,
+                        scheduledFunction);
+
+                return shouldThrottleTxn(true, innerTxnInfo, now, state);
+            }
+
+            return false;
+        }
+    }
+
+    private boolean shouldThrottleScheduleSign(
+            ThrottleReqsManager manager, TransactionInfo txnInfo, Instant now, HederaState state) {
+        final var txnBody = txnInfo.txBody();
+        if (!manager.allReqsMetAt(now)) {
+            return true;
+        }
+
+        // maintain legacy behaviour
+        final var configuration = configProvider.getConfiguration();
+
+        final boolean areLongTermSchedulesEnabled =
+                configuration.getConfigData(SchedulingConfig.class).longTermEnabled();
+        log.warn("Long term scheduling is {}.", areLongTermSchedulesEnabled ? "enabled" : "disabled");
+        if (!areLongTermSchedulesEnabled) {
+            return false;
+        } else {
+            // deeply check throttle only in the frontend throttle
+            if (throttleType != FRONTEND_THROTTLE) {
+                return false;
+            }
+
+            final var scheduledId = txnBody.scheduleSign().scheduleID();
+            final var scheduleStore = new ReadableStoreFactory(state).getStore(ReadableScheduleStore.class);
+            final var schedule = scheduleStore.get(scheduledId);
+            if (schedule == null) {
+                log.error(
+                        "Tried to throttle in the frontend throttle a ScheduleSign that does not exist! We should not get here.");
+                return true;
+            }
+
+            // only check deeply if the schedule could immediately execute
+            if (schedule.waitForExpiry()) {
+                return false;
+            }
+
+            TransactionBody innerTxn;
+            HederaFunctionality scheduledFunction;
+            try {
+                innerTxn = childAsOrdinary(schedule);
```

Comment:
I don't think we should be throttling the _child transaction_ on sign, as it is throttled on create and/or when executed.  Sign transactions generally only impact the schedule throttle.

Reply:
Hmm, I see what you're saying, so that would basically mean the whole `shouldThrottleScheduleSign` method should be reduced to something like the following:
```java
private boolean shouldThrottleScheduleSign(
            ThrottleReqsManager manager, TransactionInfo txnInfo, Instant now, HederaState state) {
        if (!manager.allReqsMetAt(now)) {
            return true;
        }
        
        final var configuration = configProvider.getConfiguration();
        final boolean areLongTermSchedulesEnabled =
                configuration.getConfigData(SchedulingConfig.class).longTermEnabled();
        if (areLongTermSchedulesEnabled) {
            log.warn("Long term scheduling is enabled, but throttling of long term schedules is not yet implemented.");
        }

        return false;
}
```
?

Also throttling the child transaction on sign is how it's done in mono, should we update it there as well? cc @tinker-michaelj

Reply:
If mono throttles the child (very odd choice IMO) we'll have to match that.